src/parser/shaman/elemental/modules/talents/ElementalBlast.js

```diff
@@ -19,9 +20,11 @@ class ElementalBlast extends Analyzer {
   constructor(...args) {
     super(...args);
     this.active = this.selectedCombatant.hasTalent(SPELLS.ELEMENTAL_BLAST_TALENT.id);
+    this.addEventListener(Events.removebuff.to(SELECTED_PLAYER), this.onRemoveBuff);
+    this.addEventListener(Events.applybuff.to(SELECTED_PLAYER), this.onApplyBuff);
   }
 
-  on_toPlayer_removebuff(event) {
+  onRemoveBuff(event) {
     if (ELEMENTAL_BLAST_IDS.includes(event.ability.guid)){
```

Comment:
could've taken the `ELEMENTAL_BLAST_IDS` constant as event listener and removed the IDs, only used in this file, nothing blocking